lms/envs/common.py

```diff
@@ -657,7 +657,7 @@
     # .. toggle_target_removal_date: None
     # .. toggle_warning: The COURSE_DISCOVERY_MEANINGS setting should be properly defined.
     # .. toggle_tickets: https://github.com/openedx/edx-platform/pull/7845
-    'ENABLE_COURSE_DISCOVERY': False,
+    'ENABLE_COURSE_DISCOVERY': True,
```

Comment:
Check out what will be impacted by this change, and see if you need to make the change in the above file or not